PRODDOCS-715: Cron UTC (#8679)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/deploy-jobs.md

- **Specific hours** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. Job runs will always be consistent between both hours and days, so if your job runs at 00:05, 12:05, and 23:05 UTC, it will run at these same hours each day. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.

### Cron schedule

To fully customize the scheduling of your job, choose the **Cron schedule** option and use cron syntax. With this syntax, you can specify the minute, hour, day of the month, month, and day of the week, allowing you to set up complex schedules like running a job on the first Monday of each month.

> **Note:** Cron schedules in <Constant name="cloud" /> use UTC and do not convert to your local timezone or adjust for daylight saving time.

Reviewer: ```suggestion
**Note:** Cron schedules in <Constant name="cloud" /> use UTC and don't convert to your local timezone or adjust for daylight saving time.
```
